catalog/task/create-github-release/0.1/create-github-release.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: create-github-release
+spec:
+  description: >-
+    Tekton task that creates a release on github.com via the GitHub API
+  params:
+    - name: repository
+      type: string
+      description: "The github repository where the release should be created"
+    - name: release_version
+      type: string
+      description: "The version string of the new release"
+    - name: githubSecret
+      type: string
+      description: "The kube secret to use to authenticate to GitHub, containing one key: token"
+    - name: content_directory
+      type: string
+      description: "The directory inside the workspace to find files for release"
+      default: "release/"
+  workspaces:
+    - name: data
+      description: The workspace where the binaries to release reside
+  results:
+    - name: url
+      type: string
+      description: URL to inspect the created release
+  steps:
+    - name: create-release-from-binaries
+      image: registry.access.redhat.com/ubi9/ubi:latest@sha256:351ed8b24d440c348486efd99587046e88bb966890a9207a5851d3a34a4dd346
+      script: |
+        #!/usr/bin/env bash
```

Comment:
If you don't use a shebang, [tekton will use a default one](https://tekton.dev/vault/pipelines-v0.44.x-lts/tasks/#running-scripts-within-steps) and also `set -e`. Without `set -e`, even if the long multiline command fails, the script will carry on, save the `url` result and be marked as successful. I think it should probably fail, so I would suggest to add `set -e`, or remove the shebang. Maybe that was actually the intention of having all the commands interconnected with `&&`? So adding the last `echo` to the chain would probably have the same effect. But just adding `set -e` and removing all the `&&` would probably be nicer.

Reply:
:+1:, updated in 129e02c.